great-idea/css/index.css

```diff
@@ -63,4 +63,201 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+* {
+	box-sizing: border-box;
+}
+.massContainer {
+	display: flex;
+	flex-direction: column;
+	margin: 0 15% 0 15%;
+
+	
+}
+
+hr {
+	color: grey;
+	width: 100%;
+	size: 5px;
+}
+
+.navContainer {
+	display: flex;
+	flex-direction: row;
+	justify-content: center;
+	width: 70%;
+	height: 60px;
+	margin-bottom: 30px;
+	position: fixed;
+	background: lavender;
+	border: 2px black;
+	border-style: none solid solid none;
+}
+
+.navBar {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-around;
+	width: 75%;
+	margin-top: 25px;
+}
+
+a {
+	text-decoration: none;
+	color: black;
+	font-style: italic;
+	align-self: space-around;
+}
+
+
+.logo {
+	display: flex;
+	justify-content: flex-end;
+	margin: 10px;
+} 
+
+
+/* ==== index.html === */
+
+.callToAction {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-evenly;
+	margin-top: 75px;
+}
+
+.iod {
+margin-top: 50px;
+text-align: center;
+border: 1px solid grey;
+}
+
+h1 {
+	font-size: 75px;
+}
+
+.button {
+	border: 1px solid black;
+	width: 120px;
+	margin-left: 29%;
+	font-size: 20px;
+	height: 40px;
+	padding: 10px;
+	
+
+	
+}
+
+
+.sectionOne {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-evenly;
+	margin: 10px;
+	
+}
+
+.middle-img {
+	display: flex;
+	justify-content: space-evenly;
+	margin: 10px;
+}
+
+.sectionTwo {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-evenly;
+	margin: 25px;
+	word-wrap: break-word
+	;
+}
+
+.footer {
+	display: flex;
+	flex-direction: column;
+	margin: 30px;
+}
+
+h4 {
+	align-self: center;
+}
+
+/* ==== end of index.html ==== */
+
+/* ==== services.html ==== */
+
+.s-header-img {
+	margin-top: 75px;
+	margin-bottom: 15px;
+}
+
+.s-services {
+	font-size: 32px;
+}
+
+.more {
+	border: 1px solid black;
+	border-radius: 8px;
+	background-color: white;
+	color: black;
+	padding: 2px;
+	
+	
+}
+
+.more:hover {
+	background-color: black;
+	color: white;
+	padding: 5px;
+	border: 1px solid white;
+	border-style: outset;
+	
+
+}
+
+.blockOne {
```

Comment:
Nice approach to the 2x3 grid on the Services page; definitely effective to use flex-wrap & item width of <50% to achieve this. Next week you'll be learning ways to have even more control of your layout on different screen sizes. :)